src/vmm/src/lib.rs

```diff
@@ -351,6 +354,25 @@ impl Vmm {
         &self.guest_memory
     }
 
+    /// Sets RDA bit in serial console
+    pub fn set_rda_pio_serial(&self) -> Result<()> {
```

Comment:
This function should do the same for `aarch64` as well not just `x86_64`.

Also the function doesn't make much sense to live on the vmm and be called at any point in the vmm lifetime, but that's less important.

Reply:
I agree with you, but this is a quick fix in order to unblock the v1.0 release, afterthat we will consume serial state from upstream